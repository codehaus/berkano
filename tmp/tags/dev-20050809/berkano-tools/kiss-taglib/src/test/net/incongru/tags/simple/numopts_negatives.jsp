<%@taglib uri="/target/classes/META-INF/taglib.tld" prefix="s"%>
From negative to positive:
<s:numericoptions start="-3" end="3"/>
With a negative step value
<s:numericoptions start="5" end="-5" step="-2"/>
