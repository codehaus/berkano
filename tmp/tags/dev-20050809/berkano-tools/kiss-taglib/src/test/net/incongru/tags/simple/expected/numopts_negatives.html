From negative to positive:
<option value="-3">-3</option>
<option value="-2">-2</option>
<option value="-1">-1</option>
<option value="0">0</option>
<option value="1">1</option>
<option value="2">2</option>
<option value="3">3</option>
With a negative step value
<option value="5">5</option>
<option value="3">3</option>
<option value="1">1</option>
<option value="-1">-1</option>
<option value="-3">-3</option>
<option value="-5">-5</option>
